wbooks/src/app/components/SearchEmpty/styles.ts

```diff
@@ -0,0 +1,24 @@
+import { StyleSheet } from 'react-native';
+import colors from '@constants/colors';
+
+export default StyleSheet.create({
+  container: {
+    alignItems: 'center',
+    backgroundColor: colors.polar,
+    flex: 1,
+    justifyContent: 'center'
+  },
+  searchIcon: {
+    height: 50,
+    marginBottom: 20,
+    width: 50
```

Comment:
Same, don't repeat => `ICON_SIZE = 50` for example

Reply:
Refactored